lib/view_model/transaction_details_view_model.dart

```diff
@@ -362,13 +362,13 @@ abstract class TransactionDetailsViewModelBase with Store {
           return setNewFee(priority: transactionPriority);
         }));
 
-    RBFListItems.add(StandardExpandableListItem(
-        title: S.current.inputs,
-        expandableItems: transactionInfo.inputAddresses ?? []));
+    if (transactionInfo.inputAddresses != null || transactionInfo.outputAddresses != null) {
+      RBFListItems.add(StandardExpandableListItem(
+          title: S.current.inputs, expandableItems: transactionInfo.inputAddresses!));
 
-    RBFListItems.add(StandardExpandableListItem(
-        title: S.current.outputs,
-        expandableItems: transactionInfo.outputAddresses ?? []));
+      RBFListItems.add(StandardExpandableListItem(
+          title: S.current.outputs, expandableItems: transactionInfo.outputAddresses!));
+    }
```

Comment:
you have an `||` in the condition, yet you assume that both won't be null and added `!` for both

I know that if we have one of them then we most probably have the other but I am talking about the concept here, and to be on the safe side, since we are not relying on our servers anyway

so the safest way would be separate them into 2 separate if conditions and each one adds the appropriate listItem

Reply:
Yes indeed, it should be '&&' instead of '||', or items should be  added separately.